ocp_resources/resource.py

```diff
@@ -334,6 +336,7 @@ def __init__(
         node_selector_labels=None,
         config_file=None,
         context=None,
+        timeout_seconds=60,
```

Comment:
Please give a meaningful name, maybe `api_call_timeout`?

Reply:
I have to keep the name, as it is expected by the api to be timeout_seconds.

Reply:
Then please add docstring, we have `timeout` and `timeout_seconds` that may confuse the user.

Reply:
I agree. I will add docstring.